src/main/rx/sbus.c

```diff
@@ -80,6 +84,7 @@ enum {
     DEBUG_SBUS_FRAME_TIME,
 };
 
+static uint32_t sbusNeedTimePreFrame = SBUS_TIME_NEEDED_PER_FRAME;
```

Comment:
I think this should rather be `sbusTimeNeededPerFrame`.

Reply:
@mikeller  yes, i will modify it